README.md

```diff
@@ -2,37 +2,36 @@
 
 [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/cpp/cpp%20-%20client%20-%20ci?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=1611&branchName=master)
 
-This repository is for active development of the Azure SDK for C++.
+This repository is for active development of the Azure SDK for C++. For consumers of the SDK we recommend visiting our [public developer docs](https://azure.github.io/azure-sdk-for-cpp) or our versioned [developer docs](https://azure.github.io/azure-sdk-for-cpp).

```

Comment:
Links are incorrect, you probably know this, but just bringing your attention in case we actually have these docs published already.

Reply:
There isn't a correct docs.microsoft.com link for us yet.  We will need to update these links when we get that home setup.  For now both links point to our reference docs